sdk/lib/core/iterator.dart

```diff
@@ -65,5 +73,18 @@ abstract class Iterator<E> {
   /// [moveNext], even if an underlying collection changes.
   /// After a successful call to `moveNext`, the user doesn't need to cache
   /// the current value, but can keep reading it from the iterator.
+  /// ```dart
+  /// final colors = ['blue', 'yellow', 'red'];
+  ///   var color = colors.iterator;
+  ///   while (color.moveNext()) {
+  ///     print(color.current);
+  ///   }
```

Comment:
looks good, but i would have used something like **colorsIterator** or something like that for naming..

Reply:
fixed